docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/test-with-postman.md

```diff
@@ -20,8 +20,8 @@ Before using `device-onvif-camera`, the user can verify the camera's functionali
 ### Import the Postman collections
 
 Download and import the following JSON files into Postman REST client tool:  
-- [onvif_camera_without_edgex_postman_collection.json](https://raw.githubusercontent.com/edgexfoundry/device-onvif-camera/{{version}}/doc/postman/onvif_camera_without_edgex_postman_collection.json)  
-- [onvif_camera_without_edgex_postman_environment.json](https://raw.githubusercontent.com/edgexfoundry/device-onvif-camera/{{version}}/doc/postman/onvif_camera_without_edgex_postman_environment.json)  
+- [onvif_camera_without_edgex_postman_collection.json](https://raw.githubusercontent.com/edgexfoundry/device-onvif-camera/{{edgexversionmain}}/doc/postman/onvif_camera_without_edgex_postman_collection.json)  
+- [onvif_camera_without_edgex_postman_environment.json](https://raw.githubusercontent.com/edgexfoundry/device-onvif-camera/{{edgexversionmain}}/doc/postman/onvif_camera_without_edgex_postman_environment.json)  
```

Comment:
Same issue with this. They are new in this release and will change to `{{edgexversion}}` when we release